config/helm/chart/default/templates/_csidriver.tpl

```diff
@@ -28,6 +28,14 @@ CSI plugin-dir path
 	{{ printf "%s/plugins/csi.oneagent.dynatrace.com/" (trimSuffix "/" (default "/var/lib/kubelet" .Values.csidriver.kubeletPath)) }}
 {{- end -}}
 
+
```

Comment:
Should we make that even configurable? Or is appending /data to the PluginDir enough?

Reply:
I am wrong - you are right